local-cli/core/index.js

```diff
@@ -70,35 +84,51 @@ const defaultRNConfig = {
         );
       });
 
-    return flatten(plugins);
+    return flatten(commands);
+  },
+
+  getPlatformConfig(): Object {
```

Comment:
Looks like this can be just a function, so that no need to call `this....` later to refer to it?

Reply:
True. I'll fix.

Reply:
#WontFix - @grabbou, we need the expose the `getPlatformConfig()` on the `RNConfig` so that it can be used later by other commands (like `link`).